TypeCobol.Test/Parser/Programs/Cobol85/ProgramEndDisorder.rdz-Nodes.txt

```diff
@@ -0,0 +1,12 @@
+Line 4[8,27] <27, Error, Syntax> - Syntax error : "END PROGRAM" is missing.
+Line 14[8,27] <37, Warning, General> - Warning: Unexpected orphan "PROGRAM END" was discarded.
```

Comment:
This is reported as an error by IBM Compiler.

Reply:
An error is now reported with the message `Unexpected orphan "PROGRAM END".`